EE9 client calls to remote EJBs fail due to feature resolution issue

Most of the remote EJB client tests in the Platform TCK currently fail with the following:

09-23-2020 22:03:04:  TRACE: in ServiceEETest.run() method
09-23-2020 22:03:04:  TRACE: Vehicle to be used for this test is:  ejb
09-23-2020 22:03:04:  TRACE: %%%%%%% isSecuredEjbClient = false
09-23-2020 22:03:05:  ERROR: Test failed
09-23-2020 22:03:05:  ERROR: javax.naming.NamingException: CWNEN1001E: The object referenced by the java:comp/env/ejb/EJBVehicle JNDI name could not be instantiated. If the reference name maps to a JNDI name in the deployment descriptor bindings for the application performing the JNDI lookup, make sure that the JNDI name mapping in the deployment descriptor binding is correct. If the JNDI name mapping is correct, make sure the target resource can be resolved with the specified name relative to the default initial context. [Root exception is com.ibm.wsspi.injectionengine.InjectionException: CWNEN0030E: The server was unable to obtain an object instance for the java:comp/env/ejb/EJBVehicle reference.  The exception message was: RemoteException occurred in server thread; nested exception is: 
	java.rmi.RemoteException: ejbRemote feature is not enabled in server process.]
	at com.ibm.ws.injectionengine.osgi.internal.naming.InjectionJavaColonHelper.newCannotInstantiateObjectException(InjectionJavaColonHelper.java:183)
	at com.ibm.ws.injectionengine.osgi.internal.naming.InjectionJavaColonHelper.getObjectInstance(InjectionJavaColonHelper.java:118)
	at com.ibm.ws.jndi.url.contexts.javacolon.internal.JavaURLContext.lookup(JavaURLContext.java:332)
	at com.ibm.ws.jndi.url.contexts.javacolon.internal.JavaURLContext.lookup(JavaURLContext.java:370)
	at org.apache.aries.jndi.DelegateContext.lookup(DelegateContext.java:149)
	at javax.naming.InitialContext.lookup(InitialContext.java:417)
	...
Caused by: java.rmi.RemoteException: ejbRemote feature is not enabled in server process.
	at com.ibm.ws.clientcontainer.remote.server.ClientSupportImpl.findEJBByInterface(ClientSupportImpl.java:370)
	at com.ibm.ws.clientcontainer.remote.server._ClientSupportImpl_Tie.findEJBByInterface(_ClientSupportImpl_Tie.java:278)
	at com.ibm.ws.clientcontainer.remote.server._ClientSupportImpl_Tie._invoke(_ClientSupportImpl_Tie.java:97)
	at org.apache.yoko.orb.OBPortableServer.ServantDispatcher.dispatch(ServantDispatcher.java:187)
	at org.apache.yoko.orb.OBPortableServer.POA_impl._OB_dispatch(POA_impl.java:1729)
	at org.apache.yoko.orb.OB.DispatchRequest_impl.invoke(DispatchRequest_impl.java:56)
	at com.ibm.ws.transport.iiop.yoko.ExecutorDispatchStrategy$1.run(ExecutorDispatchStrategy.java:42)
	at com.ibm.ws.threading.internal.ExecutorServiceImpl$RunnableWrapper.run(ExecutorServiceImpl.java:239)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)

I am using the jakartaee-9.0 convenience feature, which includes enterpriseBeansRemote-4.0, but the server auto feature for support of enterprise beans calls from remote clients is not initializing properly.

--- a/dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.enterpriseBeansRemoteClientServer-2.0.feature
+++ b/dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.enterpriseBeansRemoteClientServer-2.0.feature
@@ -5,7 +5,7 @@
 symbolicName=io.openliberty.enterpriseBeansRemoteClientServer-2.0
 visibility=private
 IBM-API-Package: com.ibm.ws.clientcontainer.remote.common;type="internal"
-IBM-Provision-Capability: osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=com.ibm.websphere.appserver.enterpriseBeansRemote-4.0))", \
+IBM-Provision-Capability: osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=io.openliberty.enterpriseBeansRemote-4.0))", \
  osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=com.ibm.websphere.appserver.clientContainerRemoteSupportCommon-1.0))"
 IBM-Install-Policy: when-satisfied
 -features=com.ibm.websphere.appserver.clientContainerRemoteSupportCommon-1.0

